Solutions/Marain.Operations.StatusClient/Marain/Operations/Client/OperationsStatus/Models/Operation.cs

```diff
@@ -135,6 +135,10 @@ public virtual void Validate()
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "Status");
             }
+            if (TenantId == null)
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, "TenantId");
+            }
             if (CreatedDateTime != null)
```

Comment:
And `is not null` (or better still `is WhateverTheTypeIs mvVal`) in this case.

Reply:
This is Autorest generated code, so best left untouched I think.